packages/react-scripts/scripts/build.js

```diff
@@ -184,16 +184,16 @@ function build(previousFileSizes) {
       }
       const build = path.relative(process.cwd(), paths.appBuild);
       console.log(`The ${chalk.cyan(build)} folder is ready to be deployed.`);
-      console.log('You may also serve it locally with a static server:');
+      console.log('You may serve it with a static server:');
       console.log();
       if (useYarn) {
-        console.log(`  ${chalk.cyan('yarn')} global add pushstate-server`);
+        console.log(`  ${chalk.cyan('yarn')} global add serve`);
       } else {
-        console.log(`  ${chalk.cyan('npm')} install -g pushstate-server`);
+        console.log(`  ${chalk.cyan('npm')} install -g serve`);
       }
-      console.log(`  ${chalk.cyan('pushstate-server')} build`);
+      console.log(`  ${chalk.cyan('serve')} -s build`);
       console.log(
-        `  ${chalk.cyan(openCommand)} http://localhost:${process.env.PORT || 9000}`
+        `  ${chalk.cyan(openCommand)} http://localhost:${process.env.PORT || 3000}`
```

Comment:
Is 3000 the default? It might be confusing for our users who already have `npm start` running on 3000 by default, and want to test their production build without stopping the devserver.